tensorflow/compiler/xla/stream_executor/device_description.h

```diff
@@ -188,7 +189,7 @@ class RocmComputeCapability {
     return {"gfx908", "gfx90a"};
```

Comment:
BF16 should also work for RDNA 3 according to [AMD's documentation](https://gpuopen.com/learn/wmma_on_rdna3/) so it should be fine to be enabled. I did work for me to add `gfx1100` there, but I'm not sure how to properly test that it's really fine.